src/NuGet.Services.EndToEnd/Support/TestSettings.cs

```diff
@@ -35,6 +35,7 @@ public enum Mode
             IReadOnlyList<string> trustedHttpsCertificates,
             string apiKey,
             string searchBaseUrl,
+            string autoCompleteUrl,
```

Comment:
That is an assumption I'd not like to hard code in these tests. 

`SearchBaseUrl` is discovered by looking for the the `SearchGalleryQueryService/` resource type in `index.json`.

The proper way to discover the autocomplete endpoint URLs is to look for `AutocompleteServiceResourceType` in `index.json`, and not assume the URL value is the same for both resource types (even though these values are the same today).